src/components/Settings/SettingsJobsCache/index.tsx

```diff
@@ -1,6 +1,12 @@
-import { PlayIcon, StopIcon, TrashIcon } from '@heroicons/react/outline';
+import {
+  PencilIcon,
+  PlayIcon,
+  StopIcon,
+  TrashIcon,
+} from '@heroicons/react/outline';
+import { PencilIcon as SolidPencilIcon } from '@heroicons/react/solid';
```

Comment:
```suggestion
import {
  PlayIcon,
  StopIcon,
  TrashIcon,
} from '@heroicons/react/outline';
import { PencilIcon } from '@heroicons/react/solid';
```
We should be consistent in using the solid pencil icon IMO, like in other places in the UI.